libs/perun/components/src/lib/groups-list/groups-list.component.html

```diff
@@ -17,26 +15,26 @@
       <ng-container matColumnDef="select">
         <th *matHeaderCellDef mat-header-cell class="align-checkbox">
           <mat-checkbox
-            (change)="$event ? masterToggle() : null"
             *ngIf="!disableHeadCheckbox"
+            (change)="$event ? masterToggle() : null"
             [aria-label]="checkboxLabel()"
             [checked]="selection.hasValue() && isAllSelected()"
             [indeterminate]="selection.hasValue() && !isAllSelected()"
             color="primary">
           </mat-checkbox>
         </th>
-        <td *matCellDef="let row" class="static-column-size align-checkbox" mat-cell>
+        <td *matCellDef="let group" class="static-column-size align-checkbox" mat-cell>
           <span
-            matTooltip="{{getCheckboxTooltipMessage(row) | translate}}"
+            *ngIf="{disable: group | disableGroupSelect: disableMembers : disableGroups : groupsToDisableCheckbox} as vm"
+            matTooltip="{{group | groupCheckboxTooltip: false | translate}}"
             [matTooltipPosition]="'above'"
-            [matTooltipDisabled]="!disableSelect(row)">
+            [matTooltipDisabled]="!vm.disable">
```

Comment:
also what is 'vm'?

Reply:
So the `*ngIf="{some: thing | pipe} as vm"`  is a way of reusing pipe results within the element. To my understanding "vm" (view model) is a common name for this. However we could use more descriptive naming.

Reply:
maybe something like vmThing would be probably more descriptive of what the VM is?